src/command_queue.rs

```diff
@@ -1307,19 +1324,19 @@ mod tests {
         println!("queue.properties(): {:X}", value);
         // assert_eq!(2, value);
 
-        // CL_VERSION_2_0 value
+        #[cfg(feature = "CL_VERSION_2_0")]
```

Comment:
As above, the call to `queue.size()` should not be an CL_VERSION_2_0 feature, therefore this change is unnecessary.